readthedocs/projects/tasks.py

```diff
@@ -1005,3 +1007,26 @@ def sync_callback(_, version_pk, commit, *args, **kwargs):
     """
     fileify(version_pk, commit=commit)
     update_search(version_pk, commit=commit)
+
+
+@app.task()
+def finish_inactive_builds():
+    delta = datetime.timedelta(hours=1, minutes=30)
```

Comment:
Lets use a `timedelta` of our base task timeout. For an example of how we add overhead to this, see:
https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/core/utils/__init__.py#L118-L128

There's a second part to this though, see below.